### ISSUE TITLE
Is there anyway to keep the underlying rest call alive ?

### ISSUE THREAD

**Issue by szd116** [knowm/XChange#3052]
This way the rest call should significantly reduce round trip delays. Thanks.

**Comment by walec51** [comment]
what to you mean by "rest call"?

or do you mean the connection?

In HTTP 1.1, all connections are considered persistent unless declared otherwise. Are you suggesting xchange does it otherwise?

**Comment by ajutzeler** [comment]
I am able to reproduce this problem with Binance.

If the interval between the API calls is less than 5 seconds, the connection is reused without any problem.

However if it is longer, the first connection is actively closed (I can intercept the tcp FIN in the trafic) by the client somehow just before the subsequent call.

To my knowledge there is no argument/property to specify the max duration of persistent connections at the level of ResCU, HTTPClient or the JVM.

I've checked my OS settings and I've got the default value of 7200 seconds.

Regarding the HTTP headers, Binance does respond with Connection: keep-alive, but does not provide any Keep-Alive header.

Any idea as where this 5 seconds value can be changed?
